tests/synpress/specs/01transactionless/walletInteraction/SwitchingNetworkTests.ts

```diff
@@ -8,172 +8,165 @@ describe('Switching from mainnet tests', () => {
   before(() => {
     cy.clearLocalStorage()
     MetamaskNetworkHandler.addARinkeby()
-    MetamaskNetworkHandler.switchToRinkebyIfNotConnected()
-  })
-  beforeEach(() => {
+    MetamaskNetworkHandler.addGnosis()
+    MetamaskNetworkHandler.switchToNetworkIfNotConnected()
     SwapPage.visitSwapPage()
     MenuBar.connectWallet()
   })
-  afterEach(() => {
-    cy.disconnectMetamaskWalletFromAllDapps()
+  beforeEach(() => {
+    cy.wait(5000)
+    MetamaskNetworkHandler.switchToNetworkIfNotConnected()
   })
+  afterEach(() => {})
   after(() => {
     cy.disconnectMetamaskWalletFromAllDapps()
-    cy.resetMetamaskAccount()
     cy.wait(500)
   })
 
   it('Should display that Ropsten network isnt supported [TC-56]', () => {
-    cy.changeMetamaskNetwork('ropsten')
+    MetamaskNetworkHandler.switchToNetworkIfNotConnected('ropsten')
     MenuBar.getUnsupportedNetworkWarning().should('be.visible')
     MenuBar.getUnsupportedNetworkPopover().should('be.visible')
   })
-  it('Should switch from unsupported network to mainnet wallet [TC-57]', () => {
-    cy.changeMetamaskNetwork('ropsten')
+  it.skip('Should switch from unsupported network to mainnet wallet [TC-57]', () => {
```

Comment:
Is this `skip` intentional? Also I would consider using different network since Ropsten is scheduled to be shutdown in Q4 2022.

Reply:
yuup , it is. All the tests will be switched to another net in another pr